src/custom-types.json

```diff
@@ -813,6 +813,14 @@
       "https://en.wikipedia.org/wiki/Processing_(programming_language)"
     ]
   },
+  "text/x-rst": {
+    "compressible": true,
+    "extensions": ["rst"],
+    "sources": [
+      "https://docutils.sourceforge.io/rst.html",
```

Comment:
I cannot find the mime type text/x-rst anyone on this page. I may be over looking it, so if you can point out where it is, please do so as a response to this comment.

Reply:
I am not sure why they have not given on the above page but this information is available on Wikipedia page. This file format is more common in the python community. It is mentioned here also https://packaging.python.org/guides/making-a-pypi-friendly-readme/.

I will try to search for other sources.